src/components/AddressLink/index.tsx

```diff
@@ -16,6 +15,17 @@ const Link = styled(SmartLink)`
   }
 `
 
+const AddressWrap = styled.div`
+  display: flex;
+  flexdirection: row;
+  grid-gap: 0.25em;
+  line-height: ${props => props.hideCopy && '1.5'};
+`
```

Comment:
@navFooh this seems to have changed some styling in the snapshots - just wanted to flag it in case i missed a class that needs to be manually added when removing the box?

Reply:
This is resolved in https://github.com/glifio/react-components/pull/440